templates/data/opensearch/index.html

```diff
@@ -0,0 +1,530 @@
+            }}
+          </div>
+        </div>
+        <p>
+          Get comprehensive, 24/7 or weekday support for Charmed OpenSearch, covering the entire solution for up to 10 years.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/data/opensearch/support">Explore Opensearch Support&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          Predictable pricing
+          <br />
+          for enterprise-grade OpenSearch
+        </h2>
+      </div>
+      <div class="col">
+        <div class="p-image-wrapper u-hide--small">
+          <div class="p-image-container">
+            {{ image (
+            url="https://assets.ubuntu.com/v1/e907127d-3.png",
+            alt="",
+            width="1800",
+            height="1014",
+            hi_def=True,
+            attrs={"class": "p-image-container__image"},
+            loading="auto|lazy"
+            ) | safe
+            }}
+          </div>
+        </div>
+        <p>
+          Charmed OpenSearch support is priced per node, per year with no per-socket or per-core limitations, and no software licence fees.
+        </p>
+        <p>
+          Run database solutions like OpenSearch with more control over your TCO. Budgeting and financial planning are straightforward and predictable.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Data solutions advisory</h2>
+      </div>
+      <div class="col">
+        <p>
+          Unlock the potential of your data. Canonical's advisory services help you design, deploy and manage your data solutions.
+        </p>
+        <hr class="is-muted" />
+        <ul class="p-list--divided">
+          <li class="p-list__item has-bullet">Platform architecture and design</li>
+          <li class="p-list__item has-bullet">Proof of Concept (PoC)</li>
+          <li class="p-list__item has-bullet">Solution deployment</li>
+          <li class="p-list__item has-bullet">Charmed OpenSearch training</li>
+        </ul>
+        <hr class="is-muted" />
+        <p>
+          <a href="https://assets.ubuntu.com/v1/4220f815-Data%20Solutions%20Advisory%20version%20_updated%20-%20v3-2.pdf">Learn more about our advisory services&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row">
+      <div class="col-6 col-medium-3">
+        <h2>Charmed OpenSearch services</h2>
+      </div>
+      <div class="col-6 col-medium-3">
+        <p>
+          Charmed OpenSearch is available self-managed or as a fully managed solution remotely delivered in your on-premise environment or cloud tenancy.
+        </p>
+      </div>
```

Comment:
Please do the same for the cards, row--50-50. This should fix the spacing as right now the text is too close to the cards
```suggestion
    <div class="row--50-50">
      <div class="col">
        <h2>Charmed OpenSearch services</h2>
      </div>
      <div class="col">
        <p>
          Charmed OpenSearch is available self-managed or as a fully managed solution remotely delivered in your on-premise environment or cloud tenancy.
        </p>
      </div>
```

Reply:
There is a nesting of rows below so this format might create issues and how does it adds more space?